Minor readme clarification (#86)
minrk/wurlitzer: README.md

stdout = out.read()
```

> Capture stdout/stderr in StringIO and forward C-level stderr to Python sys.stdout:

Reviewer: This is not quite accurate. `stderr=STDOUT` means that stderr is captured to the same stream as stdout, which is not sys.stdout in this case.

Maybe this:

```suggestion
Capture both stdout and stderr in a single StringIO object:
```
